pkgs/data/icons/whitesur-cursors/default.nix

```diff
@@ -0,0 +1,26 @@
+{ fetchFromGitHub, lib, stdenvNoCC }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "WhiteSur-cursors";
```

Comment:
```suggestion
  pname = "whitesur-cursors";
```

Reply:
@SuperSandro2000 
Thanks. I've fixed it.